frontend/src/cards/ui/SviAlert.tsx

```diff
@@ -11,13 +11,22 @@ interface SviAlertProps {
   fips: Fips;
 }
 
-export const findRating = (svi: number) => {
+export const findRating = (svi: number, map?: boolean) => {
   if (svi < 0.34) {
+    if (map) {
+      return "Low Vulnerability";
+    }
     return "low";
   }
   if (svi > 0.67) {
+    if (map) {
+      return "High Vulnerability";
+    }
     return "high";
   }
+  if (map) {
+    return "Medium Vulnerability";
+  }
```

Comment:
I think this makes it too confusing... can you leave this fn as it was but then just `uppercase()` and use a string template to add in vulnerability?

Reply:
or make a wrapper function

```
findVerboseRating(svi: number){

    return `${findRating(row.svi).toUppercase()} vulnerability`

}

```

Just trying to avoid all those extra if checks and making single functions do too many things based on boolean flags